PropertySheets/Platform.props

```diff
@@ -1,34 +1,39 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Label="Configuration">
+    <PlatformToolset>v140_xp</PlatformToolset>

```

Comment:
(in response to issue https://github.com/project64/project64/issues/649)

So far I think I might need to change this line to v120_xp, just to avoid consuming the necessary time involved with installing VS2015 when I already have 2013 on my hard disk (and probably uninstalling 2013 to save disk space).